### PR TITLE
fix: replace some plain code fences

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by Josh-Cena** [comment]
` ```plain ` is used to mark unsupported languages. For actual plain code blocks, we use no language tag.

**Comment by nschonni** [comment]
@Josh-Cena I've seen `console` also used in other repos. The reason to add these and enable MD040 is to ensure that code blocks don't get missed if they should have syntax highlighting

**Comment by Josh-Cena** [comment]
We simply don't mark code blocks with languages that don't have Prism support. The fact that `plain` exists at all (and is gracefully ignored) is due to legacy reasons, not that we intend to support it in the first place. Any other unknown language would cause a build warning. See: https://github.com/mdn/yari/blob/63936bc42c92f2254c3be03cf99cc0e6b9c557bc/build/syntax-highlight.ts#L66-L70

**Comment by nschonni** [comment]
https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#example_code_blocks

> If the highlighting that you wish to use is not listed above you should markup the code block as plain. Additional languages may be requested by following the [this process](https://github.com/orgs/mdn/discussions/170#discussioncomment-3404366).

**Comment by Josh-Cena** [comment]
Ah yes, but there's an important difference between "code that's not supported" versus "things that's actually plain text". Most of the changes I see here are actual plain text, which should not be marked with anything. We are intentionally creating a flaw for those unsupported languages so they can be fixed over time.

**Comment by bsmth** [comment]
Hi @nschonni @Josh-Cena what should we do to move forward with this one? It looks like the blocker with these changes is a decision on code blocks with a `plain` language and from what I see we have four types:

**Pseudo** (C-like?)

````
```plain
while (condition) {
  execute code block
}
```
````


**Console**

````
```plain
Promise { <state>: "pending" }
Started request…
Received response: 200
```
````

**Formal syntax** (common on CSS and SVG pages)

````
```plain
normal | <percentage>
```
````

**No language** (common on HTTP pages)

````
```plain
511 Network Authentication Required
```
````

File: files/en-us/learn/common_questions/checking_that_your_web_site_is_working_properly/index.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Grades go from A to F. Our page is just small and meets most criteria. But we ca
 
 `ping` is a useful shell tool that tests the domain name you provide and tells you if the server's responding or not:
 
-```
+```plain
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/learn/common_questions/checking_that_your_web_site_is_working_properly/index.md:150]: I see, thanks

**Comment by dipikabh** [comment]
Hello @nschonni, would you be able to resolve the conflicts before this can be merged?

**Review by dipikabh — APPROVED** [review]
Thanks for all your work here, @nschonni!

**Comment by Josh-Cena** [comment]
Sorry, I should have requested changes to block this 😅 I _do not_ want `plain` fences where the code block contains unstructured text.

**Comment by dipikabh** [comment]
Merging this now to prevent this branch from going more stale. Let's take up any more changes/feedback in subsequent PRs. Thanks everyone 🙏.

**Comment by dipikabh** [comment]
> Sorry, I should have requested changes to block this 😅 I _do not_ want `plain` fences where the code block contains unstructured text, at least for JS. `plain` does not map to any language.

Oops, sorry @Josh-Cena. Could we perhaps start a new issue/PR to get this change in?

**Comment by Josh-Cena** [comment]
@dipikabh I'm not sure what more "changes/feedback" to give in subsequent PRs—reverting all changes, or removing all `plain` tags from code blocks? Apparently there're still diverging ideas in this thread (and Yari literally says in a code comment that `plain` should be discouraged), but we just successfully established a preference via merging this PR, and I'm not sure if that sets a precedent.

**Comment by bsmth** [comment]
Hi all. The [writing guidelines say](https//developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#example_code_blocks) we should use language `plain` if it's a code block using a language not on our list of supported languages.

For unsupported languages, there was [a consensus reached](https://github.com/orgs/mdn/discussions/170#discussioncomment-3404366) in a discussion

> Unsupported languages ... must be set as plain or wait for (highlighting) PR to be accepted.

That said, I would like to make another pass regarding changing some instances of `plain` to better describe the context, like [in the comment above](https://github.com/mdn/content/pull/20030#issuecomment-1278701118) and based on the feedback from Josh.

**Pull request merged**